common/src/main/java/com/wynntils/features/user/players/PreventTradesDuelsFeature.java

```diff
@@ -14,25 +16,41 @@
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.Event;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @FeatureInfo(category = FeatureCategory.PLAYERS)
 public class PreventTradesDuelsFeature extends UserFeature {
+    private static final int FIGHTING_CUTOFF = 30; // 30 seconds
+
+    @Config
+    public boolean onlyWhileFighting = true;
```

Comment:
I think this should be off by default, or maybe have the timer be a user config. 30 seconds is quite a bit during raids, where you can finish a room pretty quickly, then be teleported to the buff room, where you might need to trade potions or other consumables to another player.

Reply:
So you're worry is that it is too high? I was just being conservative here. This feature is on by default (which I question the logic in, at least in the state it is without this PR), which will disable all trade, ever, if you hold a weapon. 

But I will make it a config (requested by both you and bolyai now), and lower it to 15 seconds. 

I'm also thinking if checking for boss bars would be a good sign of a fight in progress. I just need to differentiate them from "informative" boss bars.

Reply:
You could check for highlights nearby that correspond with a boss bar, most bosses have the glowing effect applied by the server now

Reply:
I think actually the bossbar handler can be easily extended to catch if we're currently dealing damage to a mob. I'll hold on integrating this for a while, until I've seen what I can do of that...